arch-install.bash

```diff
@@ -207,3 +288,4 @@ systemctl enable NetworkManager
 EOF
 
 umount -R /mnt
+cryptsetup close cryptroot
```

Comment:
Please configure your editor to add newlines to the end of a file.

Reply:
Will do